src/AbstractPlotting.jl

```diff
@@ -1,4 +1,17 @@
 module AbstractPlotting
+                                                                                                                                                          5.11G   12:51:45 
+################################################################################
```

Comment:
what this above?

Reply:
Comment box...I just put it there for some structure.

Reply:
Yeah, but there is some weird unicode above it?

Reply:
huh, that's really weird.  Let me check tomorrow...